lib/parseOpenGraph.js

```diff
@@ -12,7 +12,7 @@ module.exports = function ($, callback) {
     $head.find('meta').each(function(i, el) {
       var $el = $(el);
 
-      if ($el.attr('property') && $el.attr('content') && $el.attr('content').indexOf(':' > -1)) {
+      if ($el.attr('property') && $el.attr('content') && $el.attr('property').indexOf('og:') > -1) {
```

Comment:
Is this backwards compatible here?

Reply:
You were checking the indexOf `':'` in attr `'content'` instead of property. I don't know how was it working `.indexOf(':' > -1)`. Here `':' > -1' returns false. And the `false` text was searched instead of `:`.  I must be misunderstood. Correct me if I am wrong.I changed it to check for `'og:'` in `'property'` attr instead of `'content'`. The tests are all working. I guess it is backwards compatible.

Reply:
I honestly can't remember why I was checking that this way,  But so long as we are still parsing OG tags I suppose this will be fine.